azure/packages/external-controller/.eslintrc.js

```diff
@@ -7,5 +7,9 @@ module.exports = {
     "extends": [
         "@fluidframework/eslint-config-fluid"
     ],
-    "rules": {}
+    "rules": {
+        // Demoted to warning as a workaround to layer-check challenges. Tracked by:
+        // https://github.com/microsoft/FluidFramework/issues/10226
+        "import/no-extraneous-dependencies": "warn",
```

Comment:
Do you actually need to check this in. Doesn't seem like you are adding any extraneous deps.

Reply:
The details are in #10226. This rule fires because we have test-client-utils as a devDependency but it is used as a real dependency (full description in #10226 because it's somewhat complex). The lint rule correctly flags that issue.